src/dmd/dtemplate.d

```diff
@@ -476,6 +476,17 @@ extern (C++) final class Tuple : RootObject
 {
     Objects objects;
 
+    /**
+     *  Constructs a new instance.
+     *  Params:
```

Comment:
Newline before `params`

Reply:
We really should have an updated style guide for all this that is linked from the readme or the contribute documentation.